### ISSUE TITLE
Vertical Navigation - Slider Mode - back button and navigate to submenu button do not have accessibility attribute "aria-label"

### ISSUE THREAD

**Issue by paulaclemente** [lumada-design/hv-uikit-react#3357]
### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

The "back" button in the header menu and the "navigate to submenu" button in front of a menu which contains submenus should have the attribute "aria-label".

### Expected behavior 🤔

Both "back" button and  "navigate to submenu" button should have the accessibility attribute "aria-label".

### Steps to reproduce 🕹

Steps to reproduce:

1. Open Storybook
2. Go to "Vertical Navigation -> Slider Mode" example
3. Right-click on it and open the Web Inspector: inspect the _back_ button and the _navigate to submenu_ button and notice that the "aria-label" attribute is not set for any of them.

**Comment by zettca** [comment]
Hello @paulaclemente 👋 

We'll be addressing this in https://hv-eng.atlassian.net/browse/HVUIKIT-6743

**Comment by MEsteves22** [comment]
Hello 👋

Thank you for opening this issue! This was fixed on version [5.24.0](https://github.com/lumada-design/hv-uikit-react/releases/tag/%40hitachivantara%2Fuikit-react-core%405.24.0).
